CMakeLists.txt

```diff
@@ -230,7 +230,16 @@ target_include_directories(xeus-cling
                            PUBLIC
                            $<BUILD_INTERFACE:${XEUS_CLING_INCLUDE_DIR}>
                            $<INSTALL_INTERFACE:include>)
-target_link_libraries(xeus-cling PUBLIC clingInterpreter clingMetaProcessor clingUtils xeus pugixml cxxopts::cxxopts)
+target_link_libraries(xeus-cling
+                      PUBLIC
+                      xeus
```

Comment:
`xeus` needs to be linked publically as it is used in the client code in `xcpp`.